submissions/Jemikalajah-Philip/Html/homepage.html

```diff
@@ -57,7 +57,7 @@
                         <div class="col-md-3 col-sm-6">
                             <div class="product-grid">
                                 <div class="product-img">
```

Comment:
Same with this. class `product-img` isn't appropriate as the class styles your social lists also. naming is important for other users reading your code.